rednotebook/files/rednotebook-highlight-style.xml

```diff
@@ -113,6 +113,7 @@
   <style name="t2t:url" underline="single" foreground="skyblue1"/>
   <style name="t2t:link-url" foreground="skyblue1"/>
   <style name="t2t:hashtag" foreground="scarletred2"/>
+  <style name="t2t:entryReference" foreground="skyblue1"/>
```

Comment:
t2t:entryReference -> t2t:entry-reference, i.e. use lower case instead of camel case here and in the files below for consistency.

Reply:
Indeed - fixed